### ISSUE TITLE
Doorway Common App style overrides

### ISSUE THREAD

**Issue by emilyjablonski** [metrotranscom/doorway#384]
Designs: https://www.figma.com/file/xAkonVbcVCmc0jLLRcaJuA/Common-Application?type=design&node-id=496-17707&mode=design&t=OBmDTy6Dz6oXoLNo-0

After the core changes go in and have been brought over to Doorway, we need to apply design system token overrides to be able to achieve the different look and feel in Doorway in the application. The designs layout blue indicators for core changes and orange indicators for Doorway specific changes.

Most of this should hopefully be accomplishable with existing Seeds overrides. It's likely the case we made need to add some new component level overridable styles.

If we run into something that is a UIC change - if it's small, we should move forward with that now. If it's large, weigh what it would be like to do it in Seeds instead - but recognize the timeline we need for release here if we need to do a quick fix.

**Comment by sarahlazarich** [comment]
jared to take a look and break into smaller stories 
can estimate smaller tickets

**Comment by jaredcwhite** [comment]
#420 is now a blocker. Once that work is finished, we can conclude with this.

**Comment by slowbot** [comment]
@jaredcwhite Feel free to push back if any of these feel out of scope and I can create followup tickets

1. Can we update the helper text number formatting to match the filtering in the input with parenthesis? 
![image](https://github.com/metrotranscom/doorway/assets/13915/3ec14873-1f6e-4a6e-8a59-033ba639884c)

2. Can we eliminate the double padding that happens at the bottom of some form cards? 
![image](https://github.com/metrotranscom/doorway/assets/13915/e13644ef-c017-4fbc-82a4-23f8e7439c2b)

3. Can we remove this phantom link at the bottom of the demo page? 
![image](https://github.com/metrotranscom/doorway/assets/13915/7af25aa2-3acd-44dc-9b15-934fb20806cf)

4. Can we fix the contrast on this application review screen which is linked from the confirmation page? /applications/view

5. Can we align header styles and colors on the confirmation? 
![image](https://github.com/metrotranscom/doorway/assets/13915/9fdcc095-06e4-40be-aabf-831722883bb2)

6. On tablet confirmation layout loses top margin? 
![image](https://github.com/metrotranscom/doorway/assets/13915/c27d9c5c-d024-4bf4-a6d9-f269f3841634)

7. On confirmation mobile can we lose rounded corners and border on form card and go edge to edge? 
![image](https://github.com/metrotranscom/doorway/assets/13915/b8434f76-ee82-4db4-923a-498c7b093b1c)

**Comment by slowbot** [comment]
1. Can we remove this phantom link at the bottom of the demo page? 
![image](https://github.com/metrotranscom/doorway/assets/13915/7af25aa2-3acd-44dc-9b15-934fb20806cf)

**Comment by emilyjablonski** [comment]
@slowbot Would you be able to make a separate ticket for that? It's unrelated to these changes / bigger than a styling fix.

**Comment by sarahlazarich** [comment]
made a follow up https://app.zenhub.com/workspaces/bloom-5dc32d7144bd400001315dac/issues/gh/bloom-housing/bloom/3883, closing
